src/sentry/api/serializers/rest_framework/notification_action.py

```diff
@@ -29,7 +30,7 @@ def format_choices_text(choices: list[tuple[int, str]]):
 
 
 # Note the ordering of fields affects the Spike Protection API Documentation
-class NotificationActionInputData(TypedDict):
+class NotificationActionInputData(TypedDict, total=False):
```

Comment:
not my favorite :/ but `.pop`[ doesn't work ](https://mypy.readthedocs.io/en/stable/typed_dict.html#supported-operations)on non-partial TypedDicts

Reply:
jk let's get rid of the offending functions, (unsused/couldn't find any callsites). It looks like we just use `.save()` instead of` .update` or `.create`

Reply:
jk to the jk, .save() calls these functions "under the hood" 💀